omnibus-sepolia-carina.toml

```
@@ -0,0 +1,539 @@
+
+[setting.wsteth_address]
+defaultValue = "0xb82381a3fbd3fafa77b3a7be693342618240067b"
+description = "wstETH token address https://sepolia.etherscan.io/token/0xb82381a3fbd3fafa77b3a7be693342618240067b"
+
+[setting.steth_aggregator_address]
+defaultValue = "0x694AA1769357215DE4FAC081bf1f309aDC325306"
+description = "https://docs.chain.link/data-feeds/price-feeds/addresses/?network=ethereum&page=1 is actually ETH/USD."
+
+[setting.pyth_address]
+defaultValue = "0xDd24F84d36BF92C65F92307595335bdFab5Bbd21"
+description = "https://docs.pyth.network/price-feeds/contract-addresses/evm#testnets"
+
+# --- Synthetix pool settings -- #
+#
+#
+
+[setting.eth_aggregator_address]
+defaultValue = "0x694AA1769357215DE4FAC081bf1f309aDC325306"
+description = "https://docs.chain.link/data-feeds/price-feeds/addresses/?network=ethereum&page=1"
+
+[setting.weth_liquidation_reward]
+defaultValue = "<%= parseEther('10') %>"
+
+[setting.minimum_liquidity_ratio]
+defaultValue = "<%= parseEther('2') %>"
+
+[setting.account_timeout_withdraw]
+defaultValue = "86400"
+
+[setting.sc_pool_id]
+defaultValue = "1"
+
+[setting.pool_owner]
+defaultValue = "<%= settings.owner %>"
+
+[setting.wsteth_issuance_ratio_d18]
+defaultValue = "<%= parseEther('3') %>"
+
+[setting.wsteth_liquidation_ratio_d18]
+defaultValue = "<%= parseEther('1.5') %>"
+
+[setting.wsteth_liquidation_reward_d18]
+defaultValue = "<%= parseEther('0.01') %>"
+
+[setting.wstethMinDelegationD18]
+defaultValue = "<%= parseEther('0.01') %>"
+
+# --- BFP configurations  -- #
+#
+#
+
+# Collaterals
+[setting.susd_max_collateral]
+defaultValue = "<%= parseEther(String(100_000_000)) %>"
+
+[setting.weth_max_collateral]
+defaultValue = "<%= parseEther(String(1_000_000)) %>"
+
+[setting.wsteth_max_collateral]
+defaultValue = "<%= parseEther(String(1_000_000)) %>"
+
+[setting.susd_skew_scale]
+defaultValue = "0" # skew scale is not used for sUSD.
+
+[setting.weth_skew_scale]
+defaultValue = "<%= parseEther(String(1_000_000)) %>"
+
+[setting.wsteth_skew_scale]
+defaultValue = "<%= parseEther(String(1_000_000)) %>"
+
+# Global market configurations
+[setting.min_keeper_fee_usd]
+defaultValue = "<%= parseEther('15') %>"
+
+[setting.max_keeper_fee_usd]
+defaultValue = "<%= parseEther('500') %>"
+
+[setting.keeper_profit_margin_usd]
+defaultValue = "<%= parseEther('20') %>"
+
+[setting.keeper_profit_margin_percent]
+defaultValue = "<%= parseEther('0.2') %>"
+
+[setting.collateral_discount_scalar]
+defaultValue = "<%= parseEther('1') %>"
+
+[setting.min_collateral_discount]
+defaultValue = "<%= parseEther('0.02') %>"
+
+[setting.max_collateral_discount]
+defaultValue = "<%= parseEther('0.2') %>"
+
+[setting.utilization_breakpoint_percent]
+defaultValue = "<%= parseEther('0.80') %>"
+
+[setting.low_utilization_slope_percent]
+defaultValue = "<%= parseEther('0.000025') %>"
+
+[setting.high_utilization_slope_percent]
+defaultValue = "<%= parseEther('0.01') %>"
+
+# ETH market configurations
+[setting.maker_fee]
```

```suggestion
[setting.maker_fee_eth]
```
just a nit but might be worth having things suffixed/prefixed?

---

Considering this will be part of main deployment that may have other collaterals/tokens/perps?

---

One market for now, changing to add suffix if we add more.

---

Isolated deployment file that we'll revisit and refactor once testnet working